estuary/api/v1.py

```diff
@@ -115,12 +114,22 @@ def _get_partial_story(results, reverse=False):
 
     # Adding the artifact itself if it's story is not available
     if not results:
+        base_instance = estuary.utils.story.BaseStoryManager()
+        wait_times, total_wait_time = base_instance.get_wait_times([item])
         rv = {'data': [item.serialized_all], 'meta': {}}
         rv['meta']['story_related_nodes_forward'] = [0]
         rv['meta']['story_related_nodes_backward'] = [0]
         rv['meta']['requested_node_index'] = 0
         rv['meta']['story_type'] = story_manager.__class__.__name__[:-12].lower()
-        rv['meta']['wait_times'] = [0]
+        rv['meta']['wait_times'] = wait_times
+        rv['meta']['total_wait_time'] = total_wait_time
+        rv['meta']['total_processing_time'] = None
+        rv['meta']['total_lead_time'] = None
+        try:
+            rv['meta']['total_processing_time'] = base_instance.get_total_processing_time([item])
+            rv['meta']['total_lead_time'] = base_instance.get_total_lead_time([item])
```

Comment:
I just realized that the total lead time will always be 0 here since it's just a single artifact. I think you can just set it directly and move it outside of the try and except clause.